src/shared/model/Pdb.ts

```diff
@@ -12,7 +12,7 @@ export interface IPdbPositionRange {
 
 export type PdbAlignmentIndex = {
     [pdbId: string]: {
-        [chainId: string]: PdbUniprotAlignment[]
+        [chainId: string]: Alignment[]
```

Comment:
This should have been done by https://github.com/cBioPortal/cbioportal-frontend/pull/567. Seems like it slipped away type-checking because in the `PdbAnnotationAPI` it was generated as `any`: 

```JavaScript
export type PdbUniprotAlignment = any;
```